e2e/package.json

```diff
@@ -10,7 +10,7 @@
     "@types/execa": "^2.0.0",
     "@types/wait-on": "^4.0.0",
     "amphtml-validator": "^1.0.34",
-    "cypress": "^6.6.0",
+    "cypress": "^6.7.1",
```

Comment:
Because of https://github.com/cypress-io/cypress/issues/15300#issuecomment-799548183